tsdb/shard.go

```diff
@@ -2284,8 +2271,8 @@ func (fs *MeasurementFieldSet) ApplyChanges() error {
 
 // Field represents a series field. All of the fields must be hashable.
 type Field struct {
-	Name string            `json:"name,omitempty"`
```

Comment:
Why are we removing the json tagging? Just a question, not really a change comment.

Reply:
We don't JSON-ize this struct anywhere; it gets copied to a protobuf for serialization these days.